marathon/marathon.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 type Marathoner interface {
-	Apps() ([]*apps.App, error)
+	AppsWithLabel(label string) ([]*apps.App, error)
```

Comment:
Personally, I would prefer something less technical in this api, like `ManagedApps()` or `ConsulApps()` which would be implemented as private `return appsWithLabel(apps.MARATHON_CONSUL_LABEL)`. We wouldn't  have to pass that "consul" label value everywhere in the project. 
We could go further and rename `app.IsConsulApp()` to `app.IsManagedApp()` if necessary.